src/main/features/offer/routes/response.ts

```diff
@@ -0,0 +1,65 @@
+import * as express from 'express'
+import { FormValidator } from 'forms/validation/formValidator'
+import { Paths } from 'offer/paths'
+import { Paths as DashboardPaths } from 'dashboard/paths'
+import { Form } from 'forms/form'
+import { DefendantResponse } from 'offer/form/models/defendantResponse'
+import { Offer } from 'offer/form/models/offer'
+import { PartyStatement } from 'app/claims/models/partyStatement'
+import { ErrorHandling } from 'common/errorHandling'
+import User from 'idam/user'
+import { OfferGuard } from 'offer/guards/offerGuard'
+import { StatementType } from 'offer/form/models/statementType'
+import { MadeBy } from 'offer/form/models/madeBy'
+
+function redirectToDashBoardPage (res: express.Response) {
+  const user: User = res.locals.user
+  res.redirect(DashboardPaths.claimantPage.evaluateUri({ externalId: user.claim.externalId }))
+}
+function getPartyStatement (claim: any, res: express.Response): PartyStatement[] {
+  if (!claim.settlement || !claim.settlement.partyStatements) {
+    redirectToDashBoardPage(res)
```

Comment:
This is side effect. A `getPartyStatement` should only return value, not do redirects. It violates CQRS principle.

Do that logic in the place which retrieved the value from `getPartyStatement`. If undefined you can make redirect. Also guard could be possible solution as probably no point even starting processing in that router when settlement is not available.